components/StoryList/index.js

```diff
@@ -16,19 +18,23 @@ function StoryList() {
   }
 
   return (
-    <StoryListContainer>
+    <>
       <Header>My Stories</Header>
-
-      {stories.map((story) => (
-        <Link key={story.id} href={`/stories/${story.id}`}>
-          <StoryListCard>
-            <StoryCoverImage src={story.coverImage} alt={story.title} />
-            <StoryTitle>{story.title}</StoryTitle>
-            <StoryCreationDate>{story.dateCreated}</StoryCreationDate>
-          </StoryListCard>
-        </Link>
-      ))}
-    </StoryListContainer>
+      <Link href="/CreateStory">
+        <AddStoryButton type="button">Write a new Story</AddStoryButton>
+      </Link>
+      <StoryListContainer>
+        {stories.map((story) => (
+          <StyledLink key={story.id} href={`/stories/${story.id}`}>
+            <StoryListCard>
+              <StoryCoverImage src={story.coverImage} alt={story.title} />
+              <StoryTitle>{story.title}</StoryTitle>
+              <StoryCreationDate>{story.dateCreated}</StoryCreationDate>
+            </StoryListCard>
+          </StyledLink>
+        ))}
+      </StoryListContainer>
```

Comment:
```suggestion
      <ul>
        {stories.map((story) => (
        <li key={story.id} >
          <StyledLink href={`/stories/${story.id}`}>
            <StoryListCard>
              <StoryCoverImage src={story.coverImage} alt={story.title} />
              <StoryTitle>{story.title}</StoryTitle>
              <StoryCreationDate>{story.dateCreated}</StoryCreationDate>
            </StoryListCard>
          </StyledLink>
         </li>
        ))}
      </ul>
```

The naming of the StoryListContainer is a little bit confusing as these are `div` elements, but could also be a `ul` element. I would prefer to work with `ul` and `li`. See my suggestion for where I would place the elements.

Reply:
Thx, changes done... :)